generate_or_validate_json_schemas.py

```diff
@@ -223,4 +237,5 @@ def generate_or_validate_json_schemas(mode: Literal["validate", "generate"], tar
 
 
 if __name__ == "__main__":
-    generate_or_validate_json_schemas()  # pylint:disable=no-value-for-parameter
+    cli_runner = CliRunner()
+    result = cli_runner.invoke(generate_or_validate_json_schemas, ["--mode", "generate", "--target-version", "local"])
```

Comment:
```suggestion
    generate_or_validate_json_schemas()  # pylint:disable=no-value-for-parameter
```
Mein Fehler, das war noch vom lokalen Testen.